### DISCUSSION TITLE
Are there any meshtastic users in Dorset UK?

### DISCUSSION THREAD

**Discussion by camerart1** [meshtastic/Discussions#23] [General]
Hi,
First post here!
Are there any meshtastic users in Dorset UK?
[Are there any meshtastic users in Dorset UK](https://meshtastic.discourse.group/t/are-there-any-nodes-in-dorset/13849)
Cheers, Camerart

**Comment by opcon-dev** [comment]
Hi - first post for me also!

Yes - south Dorset - just received delivery of a couple of devices... :-)

**Reply by camerart1** [reply]
> Hi - first post for me also!
> 
> Yes - south Dorset - just received delivery of a couple of devices... :-)

Hi O,
I don't know if it's allowed, but I find this site difficult to navigate, could you contact me on rogerdbikeatgmail  please.
If it's not ok, let me know what your set up is once you've practiced, and see if we can send messages.
Cheers, C.

**Reply by camerart1** [reply]
Hi,
I think I saw another reply, but I can't see it in the list?
C

**Reply by DoomedFPV** [reply]
Hi,

I'm south Dorset as well.
I've just put my first outdoor node online. (Doomed Base 1, DdB1)

New to meshtastic as well just learning as I go.

**Reply by yidashcam** [reply]
Hi D,
Will you message me at the address in my first message please?
